### DISCUSSION TITLE
[LTeX] Changer de langue

### DISCUSSION THREAD

**Discussion by raylemon** [abougouffa/minemacs#141] [General]
Bonjour,
Existe-t-il un moyen de passer LTeX en français ?
J’ai essayé via un .dir-locals.el 
```
((nil .
      ((eglot-workspace-configuration
        . (:ltex-ls (:language "fr" :additionalRules (:motherTongue "fr")))))))
```
Mais en démarrant eglot, je vois bien qu’il compile le fichier elisp, mais il m’affiche continuellement les règles en anglais…

Merci !

**Comment by abougouffa** [comment]
Bonjour @raylemon 

Oui, `eglot-workspace-configuration` est très bizarre, quand je le configure comme ça, il ne prend pas la variable en compte (il n'est pas prédictible en tout cas).

J'utilise cette astuce pour le faire passer en français :

```elisp
((nil . ((eglot-workspace-configuration . eglot-ltex-workspace-config-fn)
         (eglot-ltex-language . "fr"))))
```

[La fonction `eglot-ltex-workspace-config-fn`](https://github.com/abougouffa/minemacs/blob/c770acb7b8d1e24ece65404b2a2270ad0a5cc168/modules/extras/me-eglot-ltex.el#L69-L78) est définie dans `me-eglot-ltex`, tu peux définir une fonction similaire si tu veux passer des paramètres en plus à `ltex-ls`.

**Comment by ghost** [comment]
Bonjour, j'ai une question, comment créer un repot codé en html qui figurerai sur mon profil github ? merci d'avance !

**Comment by abougouffa** [comment]
@Linux-Scratcher Si j'ai bien compris votre question, vous devez créer un repo du même nom que votre nom d'utilisateur Github (Linux-Scratcher) puis ajouter un fichier README.md dans ce repo. Ce fichier va être affiché comme couverture dans votre profil Github comme la partie entourée en rouge ici:

![image](https://github.com/abougouffa/minemacs/assets/3716399/99d75a11-2a8a-438c-a5aa-6ec7331da53e)

**Reply by ghost** [reply]
ok mais comment faire pour ajouter un ficheir dans un repos ?

**Reply by ghost** [reply]
enft c bon g réussi mrc

**Comment by ghost** [comment]
et puisque j'y suis, savez-vous comment héberger une page github ?

**Reply by abougouffa** [reply]
@Linux-Scratcher regardez sur https://pages.github.com/

**Reply by ghost** [reply]
Ok merci ^^

**Reply by ghost** [reply]
@abougouffa  je n'ai âs bien compris... suije obligé d'utiliser github desktop ?

**Reply by abougouffa** [reply]
@Linux-Scratcher 

Github Desktop n'est obligatoire. In sert just à intéragir avec les repos Git, ce qu'on peut faire avec la command `git` classique ou (d'une manière limitée) via l'interface Web.

Pour les Github pages, c'est très simple:

1. Vous devez créer un repo avec `NomD'Utilisateur.github.io` (`Linux-Scratcher.github.io`)
2. Dans le repo, ajoutez un fichier `index.html` avec ce que vous voulez,
3. Attendre une minute puis visitez le site sur https://Linux-Scratcher.github.io

**Reply by ghost** [reply]
ok j'essay de suite !!

**Reply by ghost** [reply]
Et pour finire... (jarète de vous déranger après) davez vous comment hébreger un bot discord sur github ?

**Reply by abougouffa** [reply]
@Linux-Scratcher Aucune idée !

**Reply by ghost** [reply]
PG je vais essayer de comprendre :D

**Reply by ghost** [reply]
re, désolé de te déranger encor, mais j'ai une question ... comment je fait pour créer une page sur mon site github... par exemple j'aimerais créé linux-scratcher.github.io/discord, mais quand je cré un fichié et que je vais sur la page web, ça me donne un dossié txt à télécharger...
